src/components/10-atoms/button-link/react/DemoButtonLinkClick.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { useState } from 'react';
 import AXAButtonLink from './AXAButtonLink';
 
-const DemoButtonLinkClick = () => {
+function DemoButtonLinkClick() {
```

Comment:
What motivates going back to old _function_ here? I would not use function anymore, (almost) anywhere. 

I suggest to revert here and in all other cases below.